tests/unit-tests/platforms/mesa/kms/test_display_buffer.cpp

```diff
@@ -128,6 +129,7 @@ class MesaDisplayBufferTest : public Test
     int const width{56};
     int const height{78};
     mir::geometry::Rectangle const display_area{{12,34}, {width,height}};
+    glm::mat2 const identity;
     NiceMock<MockGBM> mock_gbm;
```

Comment:
I'd be inclined to a namespace scope constant, not a member variable. (And, in the latter case, inline initialization of the member variable).